internal/e2e/rollup/BUILD.bazel

```diff
@@ -35,3 +35,47 @@ jasmine_node_test(
         "@npm//unidiff",
     ],
 )
+
+filegroup(
+    name = "bundle-outputgroups-cjs",
+    srcs = [":bundle"],
+    output_group = "cjs",
+)
+
+filegroup(
+    name = "bundle-outputgroups-umd",
+    srcs = [":bundle"],
+    output_group = "umd",
+)
+
+filegroup(
+    name = "bundle-outputgroups-es5_min_debug",
+    srcs = [":bundle"],
+    output_group = "es5_min_debug",
+)
+
+filegroup(
+    name = "bundle-outputgroups-es5_min",
+    srcs = [":bundle"],
+    output_group = "es5_min",
+)
+
+jasmine_node_test(
+    name = "test-outputgroups",
+    srcs = glob(["*.spec.js"]),
```

Comment:
This should probably be another spec file that asserts the existence of sourcemap for each flavor in addition to the js file. Existing `rollup.spec.js` only checks sourcemap for the default output.